packages/sw-lib/src/lib/error-factory.js

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Can you move this code to the top-level [`lib/`](https://github.com/GoogleChrome/sw-helpers/tree/master/lib) directory, which we're using for other modules that are shared throughout the monorepo? I can see it being useful elsewhere as well.